seastar/src/sleep.rs

```diff
@@ -59,3 +59,57 @@ pub async fn lowres_sleep_secs(duration: i32) {
 pub async fn lowres_sleep_millis(duration: i32) {
     ffi::lowres_sleep_millis(duration).await.unwrap();
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use std::time::SystemTime;
+
+    #[seastar::test]
+    async fn test_sleep() {
+        // sleep_seconds
+        let now = SystemTime::now();
+        let duration = 1;
+        sleep_secs(duration).await;
+        let elapsed = now.elapsed().unwrap().as_secs() as i32;
+        assert_eq!(duration, elapsed);
+
+        // sleep_millis
+        let now = SystemTime::now();
+        let duration = 100;
+        sleep_millis(duration).await;
+        let elapsed = now.elapsed().unwrap().as_millis() as i32;
+        assert!(elapsed >= duration / 2 && elapsed < duration * 2);
```

Comment:
I'm not sure about those. It's a bad practice in tests to expect for something to happen within a particular amout of time - the CI machines can sometimes be very slow and such assertions may lead to intermittent failures in CI.

IMO it should be fine to check that at least `duration` s/ms/ns passed, so maybe we could just stick to that?

Reply:
`Right, I changed tests to check whether `elapsed >= duration`.